### DISCUSSION TITLE
auth table in the trigger ui

### DISCUSSION THREAD

**Discussion by Broom801980** [supabase/supabase#17845] [General]
Hi All,

Trying to use Supabase for my backend auth and have run into the following issue.

I needed to select the auth table in the trigger ui. Only my public tables are available and not anything else. Is there a reason why I can't make triggers from any of the locked shemas?

Look forward to hearing from anyone that can help.

Thanks, Mark

**Comment by abdu1-wahab** [comment]
I am here for the same issue
